### PR TITLE
Readingbar

### PR DESCRIPTION
**Pull request by bhoskarpooja**
## What is the change?
Added Reading Progress Bar.

## Related issue?
 #48 

## How was it tested?
tested on localhost.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing. Yes
-   [] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change? Yes
-   [] Have you made corresponding changes to the documentation? Yes
-   [] Have you tested the code before submission? Yes
-   [] Did you lint and format your code before making the Pull Request? Yes
-   [] Have you made the GIF for the component you added? Yes

## GIF you created:
Yes

https://user-images.githubusercontent.com/72018117/122272123-d3c6c900-cefd-11eb-8a47-022c8c95f21f.mp4

### CODE REVIEW

**Comment by bhoskarpooja** [comment]
Hey Bot says these conflicts must be resolved but resole conflicts button is dissabled. there is another way to resolve?? And let me know what is conflict??

**Comment by Pranav016** [comment]
> Hey Bot says these conflicts must be resolved but resole conflicts button is dissabled. there is another way to resolve?? And let me know what is conflict??

Yes the button is disabled so even I can't resolve and it says you have resolve them on command line, pull the changes and try to resolve them on command line. Let me know if any help needed.

**Comment by theBeginner86** [comment]
@bhoskarpooja, I feel that the conflicts are arising due to changes in the file names. Especially, the `card pop up on hover ` files are causing the problem. As you changed, `Card Pop up on hover ---> Cardpopuponhover`
Try to revert only the files related to `card pop up on hover` back and then push it again. 
You may do this by fetching changes from the upstream that should be the master branch of the MIRAGE-UI repository and then resolve the conflicts as @Pranav016 stated 👍

TIP:
Always fetch the changes from upstream before pushing new changes. 🙂

File: HTML+CSS+JS/Cards/Card Pop up on hover/README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 # GIF for Task:
 
 
+<img align="center" height="500" width="800" src="https://media.giphy.com/media/9pKgy1jRHMxB06OEux/giphy.gif">
```
Comment (Pranav016) [inline HTML+CSS+JS/Cards/Card Pop up on hover/README.md:6]: This won't center align, you did the same in previous component that is why I had added a div with align= center and it got centered so remember this point

File: HTML+CSS+JS/README.md [hunk 2]
```diff
@@ -49,3 +49,6 @@
 -   [Timelines](Timelines)
 
     -   [Vertical Timeline](Timelines/VerticalTimeline)
+
+     [Reading Progress Bar](ReadingProgressBar)
```
Comment (Pranav016) [inline HTML+CSS+JS/README.md:53]: add a `-` infront of it next time. Markdown syntax for bullet points

**Review by Pranav016 — APPROVED** [review]
Remember these points for next PR
Also keep commits clean

**Pull request merged**